packages/ember-engines/tests/acceptance/routeable-engine-demo-test.js

```diff
@@ -360,15 +363,33 @@ module('Acceptance | routable engine demo', function (hooks) {
   });
 
   test('initializers run within engine', async function (assert) {
-    let stub = sinon.stub(Initializer, 'initialize');
+    if (macroCondition(!dependencySatisfies('@embroider/core', '*'))) {
+      let assetLoader = this.owner.lookup('service:asset-loader');
+      await assetLoader.loadBundle('ember-blog');
+    }
+
+    const Initializer = importSync(
+      'ember-blog/initializers/ember-blog-initializer',
+    );
+
+    let stub = sinon.stub(Initializer.default, 'initialize');
 
     await visit('/routable-engine-demo/blog/new');
 
     assert.ok(stub.calledOnce, 'Initializer ran once');
   });
 
   test('instance initializers run within engine', async function (assert) {
-    let stub = sinon.stub(InstanceInitializer, 'initialize');
+    if (macroCondition(!dependencySatisfies('@embroider/core', '*'))) {
+      let assetLoader = this.owner.lookup('service:asset-loader');
+      await assetLoader.loadBundle('ember-blog');
+    }
```

Comment:
not sure what this change accomplishes 
test seems to pass well enough without any of it

Reply:
If you remove the **if** here, it will break on CI. Could you please give a try?